pkg/agent/route/interfaces.go

```diff
@@ -51,6 +52,24 @@ type Interface interface {
 	// DeleteSNATRule should delete rule to SNAT outgoing traffic with the mark.
 	DeleteSNATRule(mark uint32) error
 
+	// InitServiceProxyConfig adds basic configuration for supporting Service.
```

Comment:
"for support Service when AntreaProxyFull is enabled" (or "AntreaProxy proxyAll")?